readme.md

```diff
@@ -27,7 +27,7 @@ Open ```http://localhost:8080/``` in your browser to see the result.
 
 ## Documentation
 
-- [Add Web Dashboard to a Vue Application](https://docs.devexpress.com/Dashboard/402486/web-dashboard/dashboard-component-for-vue/add-web-dashboard-to-a-vue-application?v=20.2)
+- [Add a Web Dashboard to a Vue Application](https://docs.devexpress.com/Dashboard/402486/web-dashboard/dashboard-component-for-vue/add-web-dashboard-to-a-vue-application?v=20.2)
 - [Dashboard Component for Vue](https://docs.devexpress.com/Dashboard/401150/web-dashboard/dashboard-component-for-vue?v=20.2)
 
 ## Examples
```

Comment:
I'm not sure about a couple of links in the list. We are already in a repository that shows an example of adding a Dashboard to a Vue.js application. This repository also contains the server side based on ASP.NET Core. Two of the links below seems to show the same content. Maybe an explanation is necessary.